bootcamp/tutorials/quickstart/hybrid_search_with_milvus.ipynb

```diff
@@ -38,7 +38,7 @@
   },
```

Comment:
<p>&nbsp;&nbsp;if len(questions) &gt; 500:&nbsp;# Skip this if you want to use the full data<strong>set</strong></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>